content/sensu-go/5.14/release-notes.md

```diff
@@ -63,18 +63,22 @@ See the [upgrade guide][1] to upgrade Sensu to version 5.14.0.
 - To facilitate discussion about feature requests, in the Web UI, feedback is directed to Discourse rather than the GitHub repository's issues page.
 - In the [web UI][80], when a user lands on a page inside a namespace that no longer exists or they do not have access to, the drawer opens to that namespace switcher to help clarify next steps.
 - Updated Go version from 1.12.3 to 1.13.1.
+- The GraphQL `putWrapped` mutation now accepts wrapped JSON with empty outer objectmeta.
 
 **FIXES:**
 
 - ([Licensed tier][79]) `sensuctl` can now create federation resources.
 - ([Licensed tier][79]) `sensuctl` on Windows can now create Postgres resources.
 - ([Licensed tier][79]) Fixed a bug that resulted in event metrics being ignored when using the Postgres store.
 - Fixed a bug that caused checks to stop executing after a network error.
+- Fixed a bug that prevented `sensuctl create` with `stdin` from working.
 - Splayed proxy checks are executed every interval (instead of every interval + interval * splay_coverage).
 - GraphQL ensures that proxy entity label and annotations are redacted.
 - Fixed a bug in the ring that prevented round robin schedules from recovering after quorum loss.
+- Fixed the generic API client's `SetTypeMeta` method. The APIGroup is now correctly configured to avoid unintended "authorization denied" errors.
 - Updated [web UI][80] so that unauthorized errors emitted while creating silences or resolving events are caught and a toast is presented to communicate what occurred.
```

Comment:
```suggestion
- Updated [web UI][80] so that unauthorized errors emitted while creating silences or resolving events are caught and a notification is presented to communicate what occurred.
```

"toast" is used internally by some to describe the style of notification, but I think we should use clearer language in these public notes